sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientProvider.java

```diff
@@ -10,7 +10,7 @@
 /**
  * An {@link HttpClientProvider} that provides an implementation of HttpClient based on Netty.
  */
-public final class ReactorNettyClientProvider implements HttpClientProvider {
+public final class NettyAsyncHttpClientProvider implements HttpClientProvider {
```

Comment:
This is a breaking change. The package and the class both are public.

Reply:
These were moved into public API earlier this month and haven't been released yet